### PR TITLE
Modified MAUI Scheduler getting started content

### PR DESCRIPTION
**Pull request by saiganesh-sakthivel**
Modified MAUI Scheduler getting started content

### CODE REVIEW

File: MAUI/Scheduler/getting-started.md [hunk 1]
```diff
@@ -8,57 +8,35 @@ documentation: ug
...
-
-4. Initialize `SfScheduler.`
-
-{% tabs %}
-{% highlight XAML hl_lines="3 5" %}
-
-<ContentPage   
-    . . .
-    xmlns:scheduler="clr-namespace:Syncfusion.Maui.Scheduler;assembly=Syncfusion.Maui.Scheduler">
+## Step 1: Create a New MAUI Project
```
Comment (PaulAndersonS) [inline MAUI/Scheduler/getting-started.md:25]: Step 1: Create a New MAUI Project

Visual Studio

Go to File > New > Project and choose the .NET MAUI App template.
Name the project and choose a location, then click Next.
Select the .NET framework version and click Create.
Visual Studio Code

Open the command palette by pressing `Ctrl+Shift+P`` and type .NET:New Project and enter.
Choose the .NET MAUI App template.
Select the project location, type the project name and press enter.
Then choose Create project

File: MAUI/Scheduler/getting-started.md [hunk 2]
```diff
@@ -8,57 +8,35 @@ documentation: ug
...
 
+## Prerequisites
+
+Before proceeding, ensure the following are set up:
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code)
 
 To get start quickly with our .NET MAUI Scheduler, you can check the below video.
 
 {% youtube "https://www.youtube.com/watch?v=Io2ElO8ORSQ" %}
```
Comment (PaulAndersonS) [inline MAUI/Scheduler/getting-started.md:23]: Add video to Top. check this PR https://github.com/syncfusion-content/maui-docs/pull/2645/files

File: MAUI/Scheduler/getting-started.md (hunk 1 above)
Comment (saiganesh-sakthivel) [reply MAUI/Scheduler/getting-started.md]: addressed

File: MAUI/Scheduler/getting-started.md (hunk 2 above)
Comment (saiganesh-sakthivel) [reply MAUI/Scheduler/getting-started.md]: addressed

File: MAUI/Scheduler/getting-started.md [hunk 3]
```diff
@@ -8,57 +8,44 @@ documentation: ug
...
-    }
-}
-
-{% endhighlight %}
-{% endtabs %}
-
-## Register the handler
-
-The `Syncfusion.Maui.Core` nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+Syncfusion.Maui.Core NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.
```
Comment (PaulAndersonS) [inline MAUI/Scheduler/getting-started.md:48]: Add link

Comment (saiganesh-sakthivel) [reply MAUI/Scheduler/getting-started.md]: addressed

**Review by PaulAndersonS — APPROVED** [review]

File: MAUI/Scheduler/getting-started.md [hunk 4]
```diff
@@ -89,14 +76,48 @@ namespace GettingStarted
 {% endhighlight %}
 {% endtabs %}
 
+## Step 4: Add a Basic Scheduler view.
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/getting-started.md:79]: remove dot in heading 
Add a Basic Scheduler view.

Comment (saiganesh-sakthivel) [reply MAUI/Scheduler/getting-started.md:79]: i have address the correction in the following pr
https://github.com/syncfusion-content/maui-docs/pull/2670/files

**Pull request merged**